src/decisionengine/framework/config/ValidConfig.py

```diff
@@ -20,13 +19,15 @@
 import _jsonnet
 
 
-def _config_from_file(config_file):
+def _config_from_file(config_file, jpaths=None):
     if os.path.getsize(config_file) == 0:
         raise RuntimeError(f"Empty configuration file {config_file}")
 
-    config_str = None
+    jpathdirs = [os.path.dirname(os.path.realpath(__file__))]
+    if jpaths is not None:
+        jpathdirs = jpaths + jpathdirs
```

Comment:
Can we add a test in for this condition?

Reply:
Yeah...will come up with a solution.